### DISCUSSION TITLE
Welcome to frida-android-hook Discussions!

### DISCUSSION THREAD

**Discussion by yazgx97** [noobpk/frida-android-hook#30] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tzippy84** [comment]
Hello!
Just found this project. Nice work! I just got into frida hooks with a Flutter app on android. I bypassed ssl pinning already with a different script, but one request still seems not to come through. So I am trying your project with the bypass-ssl option, put my burpsuite certificate to /data/local/tmp and wanted to send traffic to my burp suite proxy. But how do I provide IP and port to the --proxy option? Also, flutter apps ignore the system proxy settings in android. any way to circument this?
